docs/HISTORY.txt

```diff
@@ -5,7 +5,20 @@ Changelog
 1.11.1 (unreleased)
 -------------------
 
-- Nothing changed yet.
+- New JSON API implemented, accessible with `/upgrades-api`.
+  [jone]
+
+- Executioner: `install_upgrades_by_api_ids` was added, allowing to install
```

Comment:
this should be called `Executor` - hopefully we don't carry out death sentences :wink: